### PR TITLE
opus-tools: bump to 0.2

### PR DESCRIPTION
**Pull request by eduardoabinader**
Maintainer: @thess / @antonlacon 
Compile tested: ar71xx
Run tested: ar71xx

Description:
bump to 0.2

### CODE REVIEW

File: libs/libopusenc/Makefile [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2014-2016 OpenWrt.org
```
Comment (karlp) [inline libs/libopusenc/Makefile]: drop this entirely, or use your own copyright, with a correct date.  (Unless you _actually_ have copyright assignment with openwrt.org on file)

File: libs/opusfile/Makefile [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+   TITLE:=OPUS file library
+   URL:=http://opus-codec.org/
+endef
+
+define Package/libopusfile/description
+ The opusfile library provides seeking, decode, and playback of Opus streams in the Ogg container (.opus files)
+endef
+
+CONFIGURE_ARGS += \
+	--without-flac \
```
Comment (karlp) [inline libs/opusfile/Makefile]: does this pull in unneeded dependencies or? is it something normally provided by another library?

File: libs/libopusenc/Makefile (hunk 1 above)
Comment (eduardoabinader) [reply libs/libopusenc/Makefile]: ok, gonna provide it properly.

File: libs/opusfile/Makefile (hunk 2 above)
Comment (eduardoabinader) [reply libs/opusfile/Makefile]: yes, this avoids to pull in unneeded deps. Doing basically the same as opus-tools already does.

**Comment by eduardoabinader** [comment]
done and thanks, @karlp

**Comment by karlp** [comment]
:+1: from me, ping me again if @thess  doesn't respond in a week or so...

**Comment by thess** [comment]
@eduardoabinader - I would prefer the same maintainer for all the opus packages. It's up to you. If you want to be the maintainer of all 3 packages then replace me and Ian in the opus-tools package. Otherwise use us for the dependent libraries.

**Comment by eduardoabinader** [comment]
Thanks for the choice, @thess . I will take it then.

**Pull request merged**